rpc/src/main/proto/scalardb.proto

```diff
@@ -200,6 +200,7 @@ service DistributedStorageAdmin {
   }
   rpc GetNamespaceNames(GetNamespaceNamesRequest) returns (GetNamespaceNamesResponse) {
   }
+  rpc Upgrade(UpgradeRequest) returns (google.protobuf.Empty);
```

Comment:
Oh, I didn't know we can omit curly brackets `{}` here. But please don't omit for consistency for now:
```suggestion
  rpc Upgrade(UpgradeRequest) returns (google.protobuf.Empty) {
  }
```